cmd/cmd/logLevelUpdate.go

```diff
@@ -76,6 +76,11 @@ func printUpdateLoggerHelp() {
 }
 
 func executeUpdateLoggerCmd(loggerName, logLevel string) {
-	resp := utils.UpdateMILogger(loggerName, logLevel)
-	fmt.Println(resp)
+	resp, err := utils.UpdateMILogger(loggerName, logLevel)
+
+	if err != nil {
+		fmt.Println(utils.LogPrefixError+"Updating log level of the Logger", err)
```

Comment:
Do we have a space in "LogPrefixError"? Or, do  we need a space in between?

Reply:
We have a space in "LogPrefixError". Formatted the code in 41a92337180d73861be853107ba89b32e19d7ba1